### PR TITLE
Remove references to css styled content

### LINKED ISSUE #256: Remove or change some references to css_styled_content (9.0)
css_styled_content has been removed from the core. Maybe some of the examples can be updated.

See

* https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/Setup/Plugin/Index.html
* https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/ContentObjects/Records/Index.html
* https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/ContentObjects/LoadRegister/Index.html

Related: 

* https://github.com/TYPO3-Documentation/T3DocTeam/issues/108

### PR DESCRIPTION
**Pull request by linawolf**
resolves #256

backport to 9.5 and 10.4

### CODE REVIEW

File: Documentation/ContentObjects/Records/Index.rst [hunk 1]
```diff
@@ -296,10 +290,8 @@ accessible for the current user.
 Selection with categories
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Taken from the rendering of the "Categorized content elements"
-special menu as defined in "css_styled_content".
-
-.. code-block:: typoscript
+If you would want to display categorized content with a RECORDS obeject
```
Comment (sypets) [inline Documentation/ContentObjects/Records/Index.rst:293]: Small typo:

> obeject
object

The rest looks good, but I am not really the TypoScript expert.

Comment (linawolf) [reply Documentation/ContentObjects/Records/Index.rst]: fixed

File: Documentation/ContentObjects/LoadRegister/Index.rst [hunk 2]
```diff
@@ -17,11 +17,8 @@ The registers are processed in the reverse order. The register with the highest
 will be processed as the first, and the register with the lowest number will be processed
 as the last one. This corresponds to the stack priciple Last In – First Out (LIFO).
 
-Registers are used at many different places, e.g. css_styled_content
-uses registers to enumerate the classes of the headlines. The
-usefulness of registers is that some predefined configurations (like
-the page-content) can be used in various places but use different
-values as the values of the register can change during page rendering.
+With the advent of Fluid templating registers are used less often then
+they used to. In the core they are not being used anymore.
```
Comment (rowild) [inline Documentation/ContentObjects/LoadRegister/Index.rst:21]: How about this?
"...then they used to be. The core does not longer utilise registers at all."

File: Documentation/ContentObjects/Records/Index.rst [hunk 3]
```diff
@@ -250,22 +250,16 @@ Examples
...
-		20.0 {
-			source.field = records
-			tables = {$content.shortcut.tables}
-		}
-		...
-	}
+   page.42 = RECORDS
+   page.42 {
+      source.field = tx_examples_related_content
+		tables = tt_content
```
Comment (rowild) [inline Documentation/ContentObjects/Records/Index.rst:261]: Indentation needs to be fixed.

File: Documentation/ContentObjects/Records/Index.rst [hunk 4]
```diff
@@ -319,3 +311,28 @@ special menu as defined in "css_styled_content".
 Contrary to the previous example, in this case the :code:`conf` property
 is present and defines a very simple rendering of each content element
 (i.e. the header with a direct link to the content element).
+
+However the same can be achieved with a :ref:`cobj-fluidtemplate` and
```
Comment (rowild) [inline Documentation/ContentObjects/Records/Index.rst:315]: In this case, "However" should be followed by a comma.

**Comment by linawolf** [comment]
What is the status of this PR @sypets, @rowild can it be merged?

File: Documentation/ContentObjects/LoadRegister/Index.rst [hunk 5]
```diff
@@ -17,11 +17,8 @@ The registers are processed in the reverse order. The register with the highest
 will be processed as the first, and the register with the lowest number will be processed
 as the last one. This corresponds to the stack priciple Last In – First Out (LIFO).
 
-Registers are used at many different places, e.g. css_styled_content
-uses registers to enumerate the classes of the headlines. The
-usefulness of registers is that some predefined configurations (like
-the page-content) can be used in various places but use different
-values as the values of the register can change during page rendering.
+With the advent of Fluid templating registers are used less often then
+they used to be. In the core they are not being used anymore.
```
Comment (flicstar) [inline Documentation/ContentObjects/LoadRegister/Index.rst:21]:
```suggestion
With the advent of Fluid templating, registers are used less often then
they used to be. In the Core they are not being used anymore.
```
unsure about capitalization of core.

File: Documentation/ContentObjects/Records/Index.rst [hunk 6]
```diff
@@ -296,10 +290,8 @@ accessible for the current user.
 Selection with categories
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Taken from the rendering of the "Categorized content elements"
-special menu as defined in "css_styled_content".
-
-.. code-block:: typoscript
+If you would want to display categorized content with a RECORDS object
```
Comment (flicstar) [inline Documentation/ContentObjects/Records/Index.rst:293]:
```suggestion
If you want to display categorized content with a RECORDS object
```

**Review by flicstar — APPROVED** [review]
Nice work. Looks good.

File: Documentation/TopLevelObjects/Plugin.rst [hunk 7]
```diff
@@ -100,15 +100,14 @@ userFunc
...
-         a special situation, e.g. some styles will be output, when
-         css_styled_content is installed **and** a textpic element with an
-         image positioned alongside the text is present on the current page.
-         Most likely this will provide an acceptable display from the plugin
-         especially for this page, but should ideally be cleared and moved to
-         an external stylesheet.
+         `_CSS_PAGE_STYLE` is included only on the affected pages. Depending
+         on your configuration it will be written in an external file and included
+         on the page or directly added as inline CSS block. Compression
+         for page specific CSS also depends on the global `config.compressCss`
```
Comment (mbrodala) [inline Documentation/TopLevelObjects/Plugin.rst:106]:
```suggestion
         for page specific CSS also depends on the global :ts:`config.compressCss`
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by mbrodala — APPROVED** [review]

**Pull request merged**